spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -378,6 +384,46 @@
         end
       end
 
+      describe 'accessible_objects_function' do
+        context 'direct only' do
+          before do
+            priv_pm.add_association(color_1, creator, object_7)
+            expect_any_instance_of(PolicyMachineStorageAdapter::ActiveRecord)
```

Comment:
similar comment to above abt expects not going in befores, but i think this one is just setup so can be `allow_any_instance_of`

Reply:
(same goes for these `expect_any_instance_of..and_call_original`) in before blocks below